crates/flox-rust-sdk/src/models/environment.rs

```diff
@@ -130,11 +169,83 @@ impl<S: State> Environment<S> {
 
         Ok(())
     }
+}
+
+#[async_trait(?Send)]
+impl Environment for PathEnvironment<Original> {
```

Comment:
```suggestion
impl<S: EnvironmentState> Environment for PathEnvironment<S> {
```

The way this is currently written you can't call `build` or other `Environment` methods on `PathEnvironment<Temporary>`. We intentionally made the `TemporaryEnvironment` trait a small super-trait of `Environment` so that you can call `Environment` methods on anything that implements `TemporaryEnvironment`.